docs-src/content/using-diatheke-sdk/session/event-stream.md

```diff
@@ -335,6 +421,16 @@ stream->commandFinished(status);
 ```
 {{% /tab %}}
 
+{{% tab "Python" %}}
+``` python
+# Use the client
+client.command_finished(status)
```

Comment:
Why doesn't this need a session id?

Reply:
Because the status object contains the session id (as well as the command id and many other things).